src/tests/universal-nrp-e2e/UniversalNRP.Tests.ps1

```diff
@@ -32,13 +32,11 @@ Describe 'Validate Universal NRP' {
 
         It 'Ensure resons are properly populated' {
```

Comment:
Should there also be a test to make sure that `RuleId` is included? Seems like the below test is checking for `BaselineSettingCompliant` and `BaselineSettingNotCompliant` only.

Reply:
The actual test in in the new function in Asb.c, AsbIsValidResourceIdRuleId. If a rule id or rule name is changed from expected, the reason code will state that and fail this test in this PS script.  The NRP itself validates that the input from MOF from MC and AzP is correct and that is invoked for this test. It works quite well actually.

Reply:
Please reactivate if you still have questions. Thanks